### PR TITLE
Fix #13108: Negative integer parameter causes "Unknown alias" error w…

### LINKED ISSUE #13108: Console application: Negative integer parameter causes "Unknown alias" error
### What steps will reproduce the problem?
Create console controller with an action which has one required parameter:

    public function actionSend($chatId)
    {
        $result = \Longman\TelegramBot\Request::sendMessage([
            'chat_id'   => $chatId,
            'text'      => 'some text',
        ]);
    }

Than run this command with negative `chatId`:

    yii messenger/send -12345

### What is the expected result?

Expected result is running `actionSend()` with `$chatId` equals to `-12345`

### What do you get instead?

You'll get `Error: Unknown alias: -12345`

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.10
| PHP version      | 7.0.13
| Operating system | Ubuntu 16.0.4

### PR DESCRIPTION
**Pull request by pana1990**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  |  #13109, #13108

Fix #13108: Negative integer parameter causes "Unknown alias" error when you execute command console

### CODE REVIEW

**Review by samdark — APPROVED** [review]

**Review by cebe — APPROVED** [review]

File: framework/console/Request.php [hunk 1]
```diff
@@ -73,7 +73,11 @@ public function resolve()
                 }
             } elseif (preg_match('/^-(\w+)(?:=(.*))?$/', $param, $matches)) {
                 $name = $matches[1];
-                $params['_aliases'][$name] = isset($matches[2]) ? $matches[2] : true;
+                if (is_numeric($name) === false) {
```
Comment (dynasource) [inline framework/console/Request.php]: is_numeric already returns a boolean http://php.net/manual/en/function.is-numeric.php. No need for a comparison here

File: tests/framework/console/RequestTest.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+use yii\console\Request;
+use yiiunit\TestCase;
+
+/**
+ * @group console
+ */
+class RequestTest extends TestCase
+{
+    public function testResolve()
```
Comment (dynasource) [inline tests/framework/console/RequestTest.php]: the use of dataProvider would cleanup your test and provide consistency throughout the test environment

**Review by dynasource — CHANGES_REQUESTED** [review]

File: framework/console/Request.php (hunk 1 above)
Comment (samdark) [inline framework/console/Request.php]: `if (!is_numeric`

Comment (pana1990) [inline framework/console/Request.php]: done!

File: tests/framework/console/RequestTest.php (hunk 2 above)
Comment (pana1990) [inline tests/framework/console/RequestTest.php]: done!

**Review by SilverFire — APPROVED** [review]

**Comment by SilverFire** [comment]
Thank you!

**Pull request merged**